@here/harp-webtile-datasource/lib/WebTileDataSource.ts

```diff
@@ -26,6 +26,12 @@ export interface WebTileRenderingOptions {
      * @default false
      */
     transparent?: boolean;
+
+    /**
+     * RenderOrder for order in which to render WebTileDataSouurces
+     * @default 0
```

Comment:
We are using api-extractor(i.e. tsdoc) tags, so tit should be `@defaultValue`
https://api-extractor.com/pages/tsdoc/tag_defaultvalue/

Reply:
done